src/main/ruby/core/truffle/interop.rb

```diff
@@ -117,6 +117,24 @@ def self.object_literal(**fields)
       o
     end
     
+    def self.java_array(*array)
+      to_java_array(array)
+    end
+    
+    def self.to_java_array(array)
+      to_java_array_inner(array.to_a)
```

Comment:
I would use a primitive with `Truffle.invoke_primitive` in such a case where we don't need an extra method.